config/routes.rb

```diff
@@ -29,5 +28,13 @@
       delete 'clear'
     end
   end
+
+  get '/admin', to: 'admin#index', as: 'admin'
+
+  get 'admin/index_eads', to: 'admin#index_eads', as: 'index_eads'
+  get 'admin/index_ead', to: 'admin#index_ead', as: 'index_ead'
+
+  match "/delayed_job" => DelayedJobWeb, :anchor => false, :via => [:get, :post]
```

Comment:
its worth thinking about locking this up behind user admin role.  This (for example) is what we have for ATLA's routes

```
  authenticate :user, lambda { |u| u.admin_area? } do
    mount DelayedJobWeb, at: "/delayed_job"
  end
```

Reply:
Yes, I meant to do this. Since there are only 2 types of users, and both are able to run the index/import task, I only am protecting it via authenticated user.